.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Will users be able to select one of the two issue templates? Can they select "none" as well?

Otherwise, what about feature requests and other kind of questions?

Reply:
They should be able to select a "blank" (misc.) issue template.